src/shared/ScreenLayout.js

```diff
@@ -5,14 +5,14 @@ import { View } from "react-native";
 import styled from "styled-components/native";
 
 const ScreenLayout = (props) => {
-  const { pageTitle, btnLabel, backButton, children } = props;
+  const { pageTitle, btnLabel, backButton, children, onPressHandler } = props;
   return (
     <View>
-      {backButton}
+      {backButton()}
```

Comment:
Can you actually move
```
<Icon
        name="arrow-left"
        size={30}
        style={{ margin: 10 }}
        onPress={() => navigation.navigate(backComponentName)}
      />
```

into here and just pass in a `backComponentName` prop or something?

Reply:
Done 🍰